Dockerfile

```diff
@@ -9,14 +9,27 @@ LABEL vendor="Pegasystems Inc." \
       name="Pega Tomcat Node" \
       version=${VERSION:-CUSTOM_BUILD}
 
+
 # Creating new user and group
 
 RUN groupadd -g 9001 pegauser && \
-    useradd -r -u 9001 -g pegauser pegauser
+    useradd -r -u 9001 -g pegauser -G root pegauser
+
+RUN groupadd -g 9002 tomcat && \
+    useradd -r -u 9002 -g tomcat tomcat
 
+RUN apt-get update && \
+    apt-get install -y gosu libcap2-bin && \
+    rm -rf /var/lib/apt/lists/* && \
+    chown pegauser:root /usr/sbin/gosu && \
+    chmod o= /usr/sbin/gosu && \
+    setcap cap_setuid,cap_setgid+ep /usr/sbin/gosu
 
 ENV PEGA_DOCKER_VERSION=${VERSION:-CUSTOM_BUILD}
 
+# Limit permissions for curl
+RUN chmod 750 /usr/bin/curl
+
 # Create directory for storing heapdump
 RUN mkdir -p /heapdumps  && \
     chgrp -R 0 /heapdumps && \
```

Comment:
Is this chgrp to root group required?  Weren't we already root when we created the directory?

Reply:
ok, will strike line 36